.drone.yml

```diff
@@ -52,7 +54,7 @@ steps:
   <<: *test_common
   commands:
     - cd /app
-    - bundle exec rspec
+    - bin/ci 
```

Comment:
I like this pattern, but would almost think we should rename this to ./bin/ci-rspec so we can perhaps drop more complex things in ./bin for testing, or building perhaps?